packages/slider/src/button.vue

```diff
@@ -0,0 +1,118 @@
+<template>
+  <div
+    ref="button"
+    class="el-slider__button-wrapper"
+    :class="{ 'hover': hovering, 'dragging': dragging }"
+    :style="wrapperStyle"
+    tabindex="0"
+    @mouseenter="handleMouseEnter"
+    @mouseleave="handleMouseLeave"
+    @mousedown="onButtonDown"
+    @touchstart="onButtonDown"
+    @focus="handleMouseEnter"
+    @blur="handleMouseLeave"
+    @keydown.left="onLeftKeyDown"
+    @keydown.right="onRightKeyDown"
+    @keydown.down.prevent="onLeftKeyDown"
+    @keydown.up.prevent="onRightKeyDown"
```

Comment:
Questions why is there only `down` and `up` event gets the `prevent` modifier?

Reply:
Because when use `tab` key focus the button and press `down` and `up`, the scroll bar also will change.

Reply:
What about the  X Axis overflows? Have you tried it?

Reply:
The logic will control button's position on X Axis. I has tried it.